case-lib/lib.sh

```diff
@@ -252,6 +252,20 @@ func_lib_get_tplg_path()
     return 0
 }
 
+func_nocodec_mode()
+{
+    uname -r |grep -q "nocodec"
+}
+
+func_upload_wav_file()
+{
+    local dir=$1
+    local file=$2
+
+    find "$dir" -maxdepth 1 -type f -name "$file" -size +0 -exec cp {} "$LOG_ROOT/" \;
+    return 0
```

Comment:
The `return 0` will become a no-op when this script is converted to `set -e`. 

I recommend you remove it and you invoke the function like this below:

```shell
func_upload_wav_file "/tmp" "bat.wav.*" || true
```
This gives the caller of the function the choice.

(Cc: @RuiqingX )

Reply:
fixed